### ISSUE TITLE
Quick Connect: Double click outside of item fires event on previous selected item

### ISSUE THREAD

**Issue by robincasey** [sequelpro/sequelpro#3085]
I have 2 items in my Quick Connect pane: development and live. Live is selected. If I double click just outside of development, live gets the double click event. I might think I did not misclick and got connected to development.

**Comment by dmoagx** [comment]
Yes, if you double clicked on an empty area in the favourite list, the previously selected item would get the double click.
This has been fixed in 8f5f36 already.
